src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimize.cpp

```diff
@@ -1522,7 +1532,12 @@ void QMCFixedSampleLinearOptimize::start()
     optTarget->getConfigurations(h5FileRoot);
     optTarget->setRng(vmcEngine->getRngRefs());
     NullEngineHandle handle;
+
+    // Compute wfn parameter derivatives
+    derivative_timer_.start();
     optTarget->checkConfigurations(handle);
+    derivative_timer_.stop();
```

Comment:
Perhaps it would be better to have this moved to an issue.  I know Josh has shown me output of QMCPACK optimizations run with verbose timers on where it was impossible to determine what was taking a large part of the time.  I wonder if perhaps it would be instructive to have him post one of these input files along with the timer output so that a concrete discussion of what additional timers would best allow for a sensible understanding of the expensive parts of the algorithms.

Reply:
Removed.